src/Model/Introduction.php

```diff
@@ -20,33 +20,30 @@
  * @property string datetime
  * @property bool   blocked
  * @property bool   ignored
- *
- * @package Friendica\Model
  */
-final class Introduction extends BaseModel
+final class Introduction extends Model
```

Comment:
FollowUp idea:
Removing  `dba` calls in favor of injecting the `Introduction` model (per `__construct` override) with:
- `$this->repo`, which is the own repository for `Introduction`
- `$this->repoContact`, which is the repository for interacting with the model `Contact`

This is maybe a solution for the n:m relationship problem as well
=> we inject the repository we need inside the models we create :-)